Pacifique78/MyDiary
# #169518453 create account API with database

#### What does this PR do?
This PR contains the setup of database and adds create account endpoint

#### Description of Task to be completed
>- install pg package
>- refactor codes to use the database

#### How should this be manually tested?
>- clone the repo
>- run `npm install` and `npm run dev` to start the server
>- use either postman or any other testing tool to see if `http://localhost:4000/api/v2/auth/signup` works
>- run `npm test` to see if the tests are passing
>- Check if the Travis test are passing

#### What are the relevant pivotal tracker stories?
#169518453

#### Screenshots (if appropriate)
![Screenshot (98)](https://user-images.githubusercontent.com/50268181/68074738-3316aa80-fda7-11e9-994f-08e23386fe62.png)
![Screenshot (99)](https://user-images.githubusercontent.com/50268181/68074739-33af4100-fda7-11e9-9519-162f595dbc6a.png)
![Screenshot (100)](https://user-images.githubusercontent.com/50268181/68074740-33af4100-fda7-11e9-9f99-961086f45d7a.png)
![Screenshot (101)](https://user-images.githubusercontent.com/50268181/68074741-3447d780-fda7-11e9-9ca4-67c76205454a.png)
